DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Reports/ReportsViewModel.cs

```diff
@@ -1,18 +1,21 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
 {
+    using System;
     using System.Collections.Generic;
     using System.Linq;
     using DigitalLearningSolutions.Data.Helpers;
     using DigitalLearningSolutions.Data.Models.TrackingSystem;
 
     public class ReportsViewModel
     {
-        public ReportsViewModel(IEnumerable<PeriodOfActivity> activity)
+        public ReportsViewModel(IEnumerable<PeriodOfActivity> activity, ActivityFilterModel filterModel)
         {
             UsageStatsTableViewModel = new UsageStatsTableViewModel(activity);
+            ActivityFilterModel = filterModel;

```

Comment:
The filter will also be used in evaluation summary, so worth making the name more generic.